### PR TITLE
Avoid 'must'

### PR DESCRIPTION
**Pull request by jensmaurer**
Partially addresses cplusplus/nbballot#578

### CODE REVIEW

File: source/templates.tex [hunk 1]
```diff
@@ -138,7 +138,7 @@
 \begin{note}
 Default arguments for function templates and for member functions of
 class templates are considered definitions for the purpose of template
-instantiation\iref{temp.decls} and must obey the one-definition rule\iref{basic.def.odr}.
+instantiation\iref{temp.decls} and thus the one-definition rule\iref{basic.def.odr} applies.
```
Comment (tkoeppe) [inline source/templates.tex:141]: Please add a comma before "and".

**Review by tkoeppe — CHANGES_REQUESTED** [review]
Please fix the commit message: "tmep" should be "temp".

File: source/templates.tex (hunk 1 above)
Comment (tkoeppe) [reply source/templates.tex:141]: @jensmaurer Would you have a moment for this ^^^?

Comment (jensmaurer) [reply source/templates.tex]: Fixed.

**Comment by jensmaurer** [comment]
> Please fix the commit message: "tmep" should be "temp".

Fixed.

**Comment by tkoeppe** [comment]
Can we fold this into #6737?

**Comment by jensmaurer** [comment]
Already covered by #6737 .

**Pull request closed without merging**